assets/js/modules/search-console/components/module/ModuleOverviewWidget/Overview.js

```diff
@@ -0,0 +1,201 @@
+/**
+ * Overview component for ModuleOverviewWidget.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * External dependencies
+ */
+import each from 'lodash/each';
+import round from 'lodash/round';
+
+/**
+ * Internal dependencies
+ */
+import { calculateChange } from '../../../../../util';
+import { Grid, Row, Cell } from '../../../../../material-components';
+import ProgressBar from '../../../../../components/ProgressBar';
+import DataBlock from '../../../../../components/DataBlock';
+
+const Overview = ( { loading, data, selectedStats, handleStatsSelection } ) => {
```

Comment:
Updated @eugene-manuilov . However I notice that we don't have a hard rule about adding the appropriate propTypes. Some components have them defined, others not. Maybe we should enforce that rule by enabling it in eslint or else it can be a source of confusion. cc @aaemnnosttv